sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveExternalCatalogSuite.scala

```diff
@@ -154,4 +155,26 @@ class HiveExternalCatalogSuite extends ExternalCatalogSuite {
     catalog.createTable(hiveTable, ignoreIfExists = false)
     assert(catalog.getTable("db1", "spark_29498").owner === owner)
   }
+
+  test("SPARK-30868 throw an exception if HiveClient#runSqlHive fails") {
+    // test add jars which not exists
+    val jarPath = "file:///tmp/not_exists.jar"
+    assertThrows[QueryExecutionException](externalCatalog.client.runSqlHive(s"ADD JAR $jarPath"))
+
+    // test change to the database which doesn't exists
+    assertThrows[QueryExecutionException](externalCatalog.client.runSqlHive(
+      s"use db_not_exists"))
+
+    // test create hive table failed
+    assertThrows[QueryExecutionException](externalCatalog.client.runSqlHive(
+      s"CREATE TABLE table_not_exists(n into)"))
```

Comment:
make the comment clear that this is to test the wrong `into` type. The table name probably doesn't matter and we can use `t`.